provisioning/provisioning-device-client/src/main/java/com/microsoft/azure/sdk/iot/provisioning/device/internal/contract/http/ContractAPIHttp.java

```diff
@@ -126,7 +123,7 @@ private HttpRequest prepareRequest(
             request.setReadTimeoutMillis(timeoutInMs);
         */
 
-        request.setHeaderField(USER_AGENT, userAgentValue);
+        request.setHeaderField(USER_AGENT, "com.microsoft.azure.sdk.iot.dps.dps-device-client/1.8.5");
```

Comment:
should this be added as a hardcoded vale at line 52?

Reply:
this can be moved to a const, yes, I can change that.